CreeDictionary/CreeDictionary/templates/CreeDictionary/word-detail.html

```diff
@@ -33,19 +33,21 @@ <h1 id="head" class="definition-title">
 
     <section class="multiple-recordings definition__recordings--not-loaded" id="recordings-dropdown" data-cy="multiple-recordings">
       <p class="multiple-recordings__help-text explainer">Choose a name from the dropdown to hear the word said by the speaker.</p>
-      <select name="recordings-dropdown" data-cy="recordings-dropdown" class="multiple-recordings__dropdown">
-        <template id="template:speakerList">
-          <option><slot name="speakerName"></slot>, <slot name="speakerDialect"></slot></option>
-        </template>
-      </select>
-      <button class="multiple-recordings__action-button" data-action="play-current-recording" aria-label="Play recording" data-cy="play-selected-speaker">
-        <svg class="definition__icon speaker-playback-action-button" focusable="false">
-          <use xlink:href="#play-solid"/>
-        </svg>
-      </button>
-      <a class="multiple-recordings__action-button" data-action="learn-about-speaker" aria-label="Learn more about speaker" data-cy="learn-about-speaker" target="_blank" rel="noopener">
-        Learn more about the speaker.
-      </a>
+      <div>
+        <select name="recordings-dropdown" data-cy="recordings-dropdown" class="multiple-recordings__dropdown">
+          <template id="template:speakerList">
+            <option><slot name="speakerName"></slot>, <slot name="speakerDialect"></slot></option>
+          </template>
+        </select>
+        <button class="multiple-recordings__action-button" data-action="play-current-recording" aria-label="Play recording" data-cy="play-selected-speaker">
+          <svg class="definition__icon speaker-playback-action-button" focusable="false">
+            <use xlink:href="#play-solid"/>
+          </svg>
+        </button>
+        <a class="multiple-recordings__action-button" data-action="learn-about-speaker" aria-label="Learn more about speaker" data-cy="learn-about-speaker" target="_blank" rel="noopener">
+          Learn more about the speaker.
+        </a>
+      </div>
```

Comment:
I tried this out and the wrapper `<div>` has no effect on the cutoff; the CSS changes suffice!

```suggestion
      <select name="recordings-dropdown" data-cy="recordings-dropdown" class="multiple-recordings__dropdown">
        <template id="template:speakerList">
          <option><slot name="speakerName"></slot>, <slot name="speakerDialect"></slot></option>
        </template>
      </select>
      <button class="multiple-recordings__action-button" data-action="play-current-recording" aria-label="Play recording" data-cy="play-selected-speaker">
        <svg class="definition__icon speaker-playback-action-button" focusable="false">
          <use xlink:href="#play-solid"/>
        </svg>
      </button>
      <a class="multiple-recordings__action-button" data-action="learn-about-speaker" aria-label="Learn more about speaker" data-cy="learn-about-speaker" target="_blank" rel="noopener">
        Learn more about the speaker.
      </a>
```